<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateGameDialog</class>
 <widget class="QDialog" name="CreateGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создание игры</string>
  </property>
  <widget class="QLabel" name="lbl_Name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Description">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Описание:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Description">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Age_Restriction">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Возрастное ограничение:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Age_Restriction">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Price">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Цена:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Price">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Save">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Cancel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
